README.md

```diff
@@ -48,4 +50,27 @@ It ignores "STATE" type Singer messages.
 - Types must be arrays
 - Types cannot change
 - JSON Schema combinations such as "anyOf" and "allOf" are not supported.
-- JSON Schema $ref is not supported.
\ No newline at end of file
+- JSON Schema $ref is not supported.
+
+## Developing
+`target-postgres` utilizes [setup.py](https://python-packaging.readthedocs.io/en/latest/index.html) for package management, and [PyTest](https://docs.pytest.org/en/latest/contents.html) for testing.
+
+### DB
+The nature of `target-postgres` is integration with [PostgreSQL](https://www.postgresql.org/). Due to this, many of the tests in this repo are what are traditionally considered "integration tests".
```

Comment:
These two sentences seem not necessary. If they are kept, can we make the wording a little more plain?